doc/operator-user-guide.md

```diff
@@ -0,0 +1,358 @@
+1. Click *Catalog > Installed Operators*. From the list of *Installed Operator*s, click _3scale Operator_.
+1. Click *API Manager > Create APIManager*
+1. Create *APIManager* object with the following content.
+
+```yaml
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: example-apimanager
+spec:
+  wildcardDomain: <wildcardDomain>
+```
+
+The `wildcardDomain` parameter can be any desired name you wish to give that resolves to an IP address,
+which is a valid DNS domain. Be sure to remove the placeholder marks for your parameters: `< >`.
+
+When 3scale has been installed, a default *tenant* is created for you ready to be used,
+with a fixed URL: `3scale-admin.${wildcardDomain}`.
+For instance, when the *<wildCardDomain>* is `example.com`, then the Admin Portal URL would be:
+
+```
+https://3scale-admin.example.com
+```
+
+Optionally, you can create new tenants on the _MASTER portal URL_, with a fixed URL:
+
+```
+https://master.example.com
+```
+
+All required access credentials are stored in `system-seed` secret.
+
+### Deployment Configuration Options
+
+By default, the following deployment configuration options will be applied:
+* Containers will have [k8s resources limits and requests](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) specified. The advantage is twofold: ensure a minimum performance level and limit resources to allow external services and solutions to be allocated.
+* Internal databases will be deployed.
+* Filestorage will be based on *Persistence Volumes*, one of them requiring *RWX* access mode. Openshift must be configured to provide them when requested.
+* Mysql will be the internal relational database deployed.
+
+Default configuration option is suitable for PoC or evaluation by a customer.
+
+One, many or all of the default configuration options can be overriden with specific field values in
+the [*APIManager*](apimanager-reference.md) custom resource.
+The 3scale operator allows all available combinations whereas templates only fixed deployment profiles.
+For instance, the operator allows you to deploy 3scale in evaluation mode and external databases mode.
+Templates do not allow this specific deployment configuration. There are templates available only for the most common configuration options.
+
+#### Evaluation Installation
+Containers will not have [k8s resources limits and requests](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) specified.
+
+* Small memory footprint
+* Fast startup
+* Runnable on laptop
+* Suitable for presale/sales demos
+
+```yaml
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: example-apimanager
+spec:
+  wildcardDomain: lvh.me
+  resourceRequirementsEnabled: false
+```
+
+Check [*APIManager*](apimanager-reference.md) custom resource for reference.
+
+#### External Databases Installation
```

Comment:
Maybe here describe the DB versions required?
In this article https://access.redhat.com/articles/2798521 the required DB versions are shown for the current 3scale release. Can we reference the article? it seems to be public.

Reply:
:+1: